fixtures/uking/zq-b20-mini-spider-light.json

```diff
@@ -76,129 +74,128 @@
         },
         {
           "dmxRange": [8, 17],
-          "type": "Effect",
-          "effectName": "Effect 1"
+          "type": "Generic",
+          "comment": "Effect 1"
```

Comment:
Why did you change these capabilities from the correct "Effect" to the much less specific "Generic"? :sweat_smile:

Reply:
You know... I really am not sure what happen there. I don't remember making that change, but this was also the fixture I was using to learn the OFL fixture format so that must have been one of my "learn experiences" 😆 

I will get the fixed up